DependencyInjection/Configuration.php

```diff
@@ -51,7 +51,7 @@ public function getConfigTreeBuilder()
                                         if (!isset($features[$featureName])) {
                                             throw new InvalidConfigurationException("Uknown feature: {$featureName}");
                                         }
-                                        $featureMask += isset($features[$featureName]) ? $features[$featureName] : 0;
+                                        $featureMask &= $features[$featureName];
```

Comment:
> isset() is not required here, you are already checking it above.

Yes, agreed.

>Mask should be built with bitwise and operator, not + operator, right?

In general there is no difference between bitwise OR and mathematical addition. But you change is wrong: to combine feature mask you should use bitwise or operator `|`: 

```php
$featureMask = A | B | C;
// $featureMask = A & B & C; will be equal to 0
```